### PR TITLE
Merge into us-metric-configuration

### PR DESCRIPTION
**Pull request by rkhatta1**
Designed the basic UI.

### CODE REVIEW

File: taigaProject/frontend/src/components/Hero.js
Comment (rnolas96) [inline taigaProject/frontend/src/components/Hero.js:1]: Do we need this component?

Comment (rnolas96) [reply taigaProject/frontend/src/components/Hero.js:1]: After discussion, we have decided that we need this to view the team members. The idea is that, if a TA or the professor views the project, the team members associated with the project is clearly visible.

**Comment by rnolas96** [comment]
Has this code been fully tested on your system? @rkhatta1

**Comment by rkhatta1** [comment]
> Has this code been fully tested on your system? @rkhatta1

Yes, it has. All the buttons and links are working well.

**Pull request merged**